src/handlers/gestures/flingGesture.ts

```diff
@@ -13,11 +13,21 @@ export class FlingGesture extends BaseGesture<FlingGestureHandlerEventPayload> {
     this.handlerName = 'FlingGestureHandler';
   }
 
+  /**
+   * Determine exact number of points required to handle the fling gesture.
+   * @param pointers
+   */
   numberOfPointers(pointers: number) {
     this.config.numberOfPointers = pointers;
     return this;
   }
 
+  /**
+   * Expressed allowed direction of movement.
+   * Expected values are exported as constants in the Directions object.
```

Comment:
Would be nice to mention that multiple directions at once can be recognized.

Reply:
Done in [4de93ca](https://github.com/software-mansion/react-native-gesture-handler/pull/2874/commits/4de93ca0ef7f3048ba5c7ae1aba35ff3d4f9ec92)